lib/msf/base/sessions/command_shell.rb

```diff
@@ -235,35 +235,35 @@ def cmd_sessions_help
   end
 
   def cmd_sessions(*args)
-    if args.length.zero? || args[0].to_i <= 0
-      # No args
+    if args.length != 1
+      print_status "Wrong number of arguments expected: 1, received: #{args.length}"
       return cmd_sessions_help
     end
 
-    if args.length == 1 && (args[1] == '-h' || args[1] == 'help')
-      # One arg, and args[1] => '-h' '-H' 'help'
+    if args[0] == '-h' || args[0] == '--help'
       return cmd_sessions_help
     end
 
-    if args.length != 1
-      # More than one argument
+    session_id = args[0].to_i
+    if session_id <= 0
```

Comment:
Could we check if this works the same as before when provided with negative session ids?

Reply:
@dwelch-r7 Is there anything actionable here?

Reply:
No, the previous implementation didn't accept negative session IDs so it's a nice to have but I don't think we should block the PR on it